maintenance_page/manifests/maintenance/deployment_maintenance.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apply-for-qts-maintenance
+  labels:
+    app: apply-for-qts-maintenance
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: apply-for-qts-maintenance
+  template:
+    metadata:
+      labels:
+        app: apply-for-qts-maintenance
+    spec:
+      nodeSelector:
+        teacherservices.cloud/node_pool: applications
+        kubernetes.io/os: linux
+      containers:
+        - name: apply-for-qts-maintenance
+          image: ghcr.io/dfe-digital/apply-for-qualified-teacher-status-maintenance:dummy-tag
+          ports:
+            - containerPort: 8080
+          resources:
+            requests:
+              memory: 64M
+              cpu: 0.1
+            limits:
+              memory: 64M
+              cpu: 1
```

Comment:
this file (deployment_maintenance.yml) is built by failover.sh, so it can be deleted from this PR

Reply:
that slipped in to my commit